backend/windmill-worker/src/go_executor.rs

```diff
@@ -215,12 +210,11 @@ func Run(req Req) (interface{{}}, error){{
         )
         .await?;
 
-        if let Some(bin_path) = bin_path.clone() {
-            tokio::fs::copy(format!("{job_dir}/main"), &bin_path).await?;
-            logs.push_str(&format!("write cached binary: {}\n", bin_path));
-        }
+        create_dir(&bin_path).await?;
```

Comment:
We're in the `if !bin_exists` clause here, so we need to create the folder before we can copy the binary to it